mantidproject/mantid
# Reflectometry GUI: Autoprocess doesn't start when not possible

**Description of work.**
Previously AutoProcess would attempt to run no matter if it is successful to login or has a search arguement



**To test:**
- Open Workbench or Mantidplot
- Open ISIS Reflectometry GUI
- Click AutoProcess (it should tell you to enter search param)
- Give the search param 1120015
- Click AutoProcess again
- It should prompt you for login details to the catalog, give some incorrect information
- The auto processing progress bar should stop and the process shouldn't be running



Re #26025. 


BUG FIX FOR UNRELEASED STUFF

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
